plantclef/workflows/baseline.py

```diff
@@ -16,8 +16,10 @@
 class ProcessBase(luigi.Task):
     input_path = luigi.Parameter()
     output_path = luigi.Parameter()
-    should_subset = luigi.Parameter(default=False)
+    should_subset = luigi.BoolParameter(default=False)
     num_partitions = luigi.IntParameter(default=500)
+    sample_id = luigi.OptionalIntParameter(default=None)
+    num_sample_id = luigi.IntParameter(default=10)
 
     def output(self):
```

Comment:
In the output, we should instead check for the success file in the output_path's data directory that's generated by spark. This way we won't skip this section. The logic needs to follow the logic the section where we write things out.